package.json

```diff
@@ -112,10 +109,6 @@
           }
         ]
       }
-    },
-    "sql": {
-      "transitive_localized_views" : false,
-      "native_hana_associations" : false
```

Comment:
We need to keep this config as I think the default is still `true`.

Reply:
true, I'll revert.